tests/utils/test_auth.py

```diff
@@ -0,0 +1,84 @@
+import unittest
+from unittest.mock import patch
+
+from flask import request
+from firebase_admin.auth import RevokedIdTokenError, InvalidIdTokenError, verify_id_token
+from werkzeug import exceptions as e
+
+from stronk import constants as c
+from stronk.utils import auth
+
+
+@patch("stronk.utils.auth.request")
```

Comment:
I think this would be the better approach because it explicitly indicates we're mocking just the `request` which is all we need. The `request` is also explicitly imported into the `auth` namespace which is also explicitly seen in the `auth.py`. Thoughts?